certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2953 @@
+          },
+          "\"Content-Type\"": {
+            "description": "\"Content-Type\"",
+            "type": "string"
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Webhooks"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "201": {
+            "description": "Returns the newly created item",
+            "schema": {
+              "$ref": "#/definitions/WebhookResponseResponse"
+            },
+            "examples": {
+              "application/json": {
+                "data": {
+                  "id": "3d2d560f-2c5b-41ee-aaef-9c5f5b1d5292",
+                  "eventTypes": [
+                    "Project:Created",
+                    "Project:Updated"
+                  ],
+                  "headers": [
+                    {
+                      "name": "Authorization",
+                      "value": "Bearer token_value"
+                    },
+                    {
+                      "name": "custom_header",
+                      "value": "header_value"
+                    }
+                  ],
+                  "httpVerb": "POST",
+                  "url": "https://example.com"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "If the item is not valid",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "If not authorized",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "summary": "Webhook Trigger",
+        "description": "When a operation is performed on an entity in PPM Express",
+        "operationId": "Webhook_Trigger",
+        "x-ms-trigger": "single",
+        "x-ms-trigger-hint": "To see it work, perform any specified action in PPM Express",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "eventTypes": {
+                  "x-ms-summary": "Event Types",
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "enum": [
+                      "Task:Created",
+                      "Task:Updated",
+                      "Task:Deleted",
+                      "Idea:Created",
+                      "Idea:Updated",
+                      "Idea:Deleted",
+                      "Project:Created",
+                      "Project:Updated",
+                      "Project:Deleted",
+                      "KeyDate:Created",
+                      "KeyDate:Updated",
+                      "KeyDate:Deleted"
+                    ],
+                    "title": "",

```

Comment:
@vsorokin-ppmexpress empty properties are not allowed, they can be removed unless they're required. Please provide a value or remove it.